internal/log_analysis/log_processor/parsers/cloudflarelogs/access.go

```
@@ -0,0 +1,224 @@
+	EdgeResponseStatus     pantherlog.Int64  `json:"EdgeResponseStatus" description:"HTTP response status code returned to browser"`
+	Kind                   pantherlog.String `json:"Kind" description:"The kind of event, currently only possible values are: firewall"`
+	MatchIndex             pantherlog.Int64  `json:"MatchIndex" description:"Rules match index in the chain"`
+	Metadata               interface{}       `json:"Metadata" description:"Additional product-specific information. Metadata is organized in key:value pairs. Key and Value formats can vary by Cloudflare security product and can change over time"`
+	OriginResponseStatus   pantherlog.Int64  `json:"OriginResponseStatus" description:"HTTP origin response status code returned to browser"`
+	OriginatorRayID        pantherlog.String `json:"OriginatorRayID" panther:"trace_id" description:"The RayID of the request that issued the challenge/jschallenge"`
+	RayID                  pantherlog.String `json:"RayID" panther:"trace_id" description:"The RayID of the request"`
+	RuleID                 pantherlog.String `json:"RuleID" description:"The Cloudflare security product-specific RuleID triggered by this request"`
+	Source                 pantherlog.String `json:"Source" description:"The Cloudflare security product triggered by this request"`
+}
+
+// NanosOrSecondsDecoder tries to guess the format of the timestamp (seconds or nanoseconds since epoch)
+// before parsing it to time.Time.
+//
+// To decide whether the timestamp is in seconds or nanoseconds, ff the parsed number
+// is greater than the current timestamp in seconds, we assume the parsed one is in nanoseconds.
+type NanosOrSecondsDecoder struct{}
+
+var _ = (tcodec.TimeDecoder)(&NanosOrSecondsDecoder{})
+
+func (d *NanosOrSecondsDecoder) DecodeTime(iter *jsoniter.Iterator) time.Time {
+	n := iter.ReadUint64()
+	// Check for error from ReadUint64()
+	if iter.Error != nil && iter.Error != io.EOF {
+		return time.Time{}
+	}
```

Shouldn't we report an error in this case? Same error handling as in here: https://github.com/panther-labs/panther/blob/b60ee537f738d86dcfd191a9aa5d76329d649bef/internal/log_analysis/log_processor/pantherlog/tcodec/tcodec.go#L164-L168